projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -126,10 +128,17 @@ public function add_upgrades_menu() {
 		}
 
 		if ( ! $menu_exists ) {
-			add_menu_page( __( 'Upgrades', 'jetpack' ), __( 'Upgrades', 'jetpack' ), 'manage_options', 'paid-upgrades.php', null, 'dashicons-cart', 4 );
+			$site_upgrades = '%1$s<span class="inline-text">%2$s</span>';
+			$site_upgrades = sprintf(
+				$site_upgrades,
+				__( 'Upgrades', 'jetpack' ),
+				$plan
+			);
+
+			add_menu_page( __( 'Upgrades', 'jetpack' ), $site_upgrades, 'manage_options', 'paid-upgrades.php', null, 'dashicons-cart', 4 );
```

Comment:
I think we could use `$this->update_menu( 'paid-upgrades.php', null, $site_upgrades, null, null, 4 );` 🤔 .

Reply:
Can you elaborate on this? It doesn't seem working since this menu doesn't already exist.

Reply:
Oh, my bad. I was under impression we had it. Sorry about that!

Reply:
No worries! Is there a chance you know more about https://github.com/Automattic/jetpack/pull/19574#issuecomment-823431368 ?